WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -76,7 +76,7 @@ extension XCTestCase {
         static let tag = "tag \(Date().toString())"
     }
 
-    public func removeApp(_ appName: String = "WordPress", app: XCUIApplication = XCUIApplication()) {
+    public func removeApp(_ appName: String = XCUIApplication().label, app: XCUIApplication = XCUIApplication()) {
```

Comment:
TIL about `XCUIApplication` `label`. I couldn't find it in the [`XCUIApplication` docs](https://developer.apple.com/documentation/xctest/xcuiapplication) or in its superclass [`XCUIElement`](https://developer.apple.com/documentation/xctest/xcuielement).

Inspecting the source showed it's actually part of [`XCUIElementAttributes`](https://developer.apple.com/documentation/xctest/xcuielementattributes).